app/viewers/embed/viewer/common_viewer.rb

```diff
@@ -34,9 +34,12 @@ def external_url
       # Creates a file url for stacks
       # @param [String] title
       # @param [Boolean] download
+      # @param [Boolean] allow_literal_slashes
       # @return [String]
-      def file_url(title, download: false)
-        uri = URI.parse("#{stacks_url}/#{ERB::Util.url_encode(title)}")
+      def file_url(title, download: false, allow_literal_slashes: false)
```

Comment:
in which cases would we not want to allow literal slashes? can you add this to the inline documentation above? Do we need to keep this as an argument?

Reply:
@jcoyne 💬 
> in which cases would we not want to allow literal slashes? 

That is the current behavior, so apparently all cases except the one we're adding for file-type objects? I don't really know. 

> can you add this to the inline documentation above?

Happy to add a blurb about why this arg was added, you bet.

> Do we need to keep this as an argument?

Good question, and I don't know. My intent here is to add a small seam to support the behavior needed for these filenames-with-path-info-in-them, without breaking anything else. If y'all think it's safe to move to literal slashes everywhere, I can make the change of course. That seems like it'd add significantly to the testing we'll have to do, and I'll need to lean on y'all for help with that.

Reply:
I would rather change it to accept literal slashes in all cases an repeat the testing. While that is a greater burden now, over the long run it will be less complexity to support.

Reply:
@jcoyne OK, that's fine with me, and I'm happy to re-run the test with the ~7-8 test objects in stage.

Reply:
@jcoyne Mind having another look?